hw_openqa.ipynb

```diff
@@ -65,7 +65,8 @@
     "    1. [Few-shot OpenQA [2 points]](#Few-shot-OpenQA-[2-points])\n",
     "    1. [Answer scoring [2 points]](#Answer-scoring-[2-points])\n",
     "    1. [Your original system [3 points]](#Your-original-system-[3-points])\n",
-    "1. [Bake-off [1 point]](#Bake-off-[1-point])"
+    "1. [Bake-off [1 point]](#Bake-off-[1-point])\n",
+    "1. [Submission Instruction](#Submission-Instruction)"
```

Comment:
Please change this title to "Submission instructions" (note the capitalization and pluralization).

Reply:
Updated, PTAL.